tests/chimp-config.js

```diff
@@ -112,7 +112,7 @@ module.exports = {
 		timeout: 40000,
 		reporter: 'min',
 		slow: 100,
-		retries: 3,
+		//retries: 3,
```

Comment:
With this setting on, some tests were acting strange like retrying successful tests.